Library/Formula/git-big-picture.rb

```diff
@@ -0,0 +1,18 @@
+require 'formula'
+
+class GitBigPicture < Formula
+  homepage 'https://github.com/esc/git-big-picture'
+  url 'https://github.com/esc/git-big-picture/archive/v0.9.0.tar.gz'
+  sha1 'd0265f32bcb9e6c1b7991e470e621c1931ee6790'
+
+  depends_on 'graphviz' => :recommended
+  depends_on 'git'
+
+  def install
+    bin.install 'git-big-picture'
+  end
+
+  test do
+    system "#{bin}/git-big-picture", "-h"
```

Comment:
Perhaps create a Git repository in the current (temporary) directory, make a commit and run `git-big-picture -o our-project.png`